infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesInternalRuntime.java

```diff
@@ -634,6 +638,7 @@ protected void listenEvents() throws InfrastructureException {
   private void watchLogsIfDebugEnabled(Map<String, String> startOptions)
       throws InfrastructureException {
     if (LogWatcher.shouldWatchLogs(startOptions)) {
+      logWatcherMetrics.workspaceStartedInStartDebugMode();
```

Comment:
can we base counter logic on Events? In this way, we will need to inject additional classes to KubernetesInternalRuntime.java.

Reply:
done